src/blocks/toggleSwtich/ToggleSwitch.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import styled from 'styled-components';
+import * as Switch from '@radix-ui/react-switch';
+
+import { Box } from '../box';
+import { Text } from '../text';
+
+export type ToggleSwitchProps = {
+  /* Label for the toggle switch */
+  label?: string;
+  /* Description for the toggle switch */
+  description?: string;
+  /* Sets toggle switch as disabled */
+  disabled?: boolean;
+  /* Render the toggle before text contents */
+  leadingToggle?: boolean;
+  /* Sets toggle switch to checked */
+  checked: boolean;
+  /* Function invoked when toggle switch is clicked*/
+  onCheckedChange: (checked: boolean) => void;
+};
+
+const StyledToggleRoot = styled(Switch.Root)`
+  width: 38px;
+  height: 20px;
+  padding: 3px;
+  background-color: var(--components-toggle-switch-background-unselected, #c4cbd5);
```

Comment:
```suggestion
  background-color: var(--components-toggle-switch-background-unselected);
```
Let's not use the fallback. Don't think it's required.